spec/factories/finance/milestones.rb

```diff
@@ -7,5 +7,6 @@
     start_date { Time.zone.now - 1.month }
     milestone_date { Time.zone.now + 1.month }
     payment_date { Time.zone.now + 2.months }
+    schedule
```

Comment:
Opinion: not convinced milestones factory should create rogue schedules. I'd rather to the test creates a meaningful schedule and takes the milestone it needs. Make the test ever so slightly more readable and self explanatory.

Reply:
agreed, i've removed this now